.github/workflows/doc-link.yml

```diff
@@ -12,3 +12,12 @@ jobs:
         repo-token: ${{ secrets.GITHUB_TOKEN }}
         artifact-path: 0/docs/build/html/index.html
         circleci-jobs: document
+  visualization-regression-tests-link:
+    runs-on: ubuntu-latest
+    if: "github.event.context == 'ci/circleci: visual-regression-tests'"
+    steps:
+    - uses: larsoner/circleci-artifacts-redirector-action@master
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        artifact-path: 0/visual_regression_tests/public/index.html
+        circleci-jobs: visual-regression-tests
```

Comment:
According to @toshihikoyanase, the link seems not to be listed on the status area of this PR until merged to master branch, since the trigger condition of this workflow is `on: status`.

> Note: This event will only trigger a workflow run if the workflow file is on the default branch.
> https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#status